app/routers/notes/notes.py

```diff
@@ -0,0 +1,60 @@
+from fastapi.param_functions import Depends
+from sqlalchemy.orm import Session
+from app.database.schemas import NoteDB, NoteSchema
+from app.dependencies import get_db
+from app.routers.notes import crud
+from fastapi import APIRouter
+
+router = APIRouter(
+    prefix="/notes",
+    tags=["notes"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.post("/", response_model=NoteDB, status_code=201)
+async def create_note(payload: NoteSchema, session: Session = Depends(get_db)):
+    note_id = await crud.post(session, payload)
+
+    response_object = {
+        "id": note_id,
+        "title": payload.title,
+        "description": payload.description,
+    }
+    return response_object
+
+
+# @router.get("/")
```

Comment:
please remove

Reply:
@ivarshav  - I didn't understand you - remove what?

Reply:
Do you need the commented out code?

Reply:
Done